tosam

```diff
@@ -2,4 +2,4 @@
 
 SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
 
-python3 ${SCRIPT_DIR}/src/tosam.py $@
+python ${SCRIPT_DIR}/src/tosam.py $@
```

Comment:
You are playing with fire here. I see why you might want to do it, but the tests are not run on your machine but on a virtual ubuntu on GitHub, and it might have a `python` that is Python 2.x, and then the tests will fail. Looks like it didn't here, though. Generally, though, you are safer if you copy the scripts and tests than if you modify them. If you change them, they might work on your own machine but will fail when you push to GitHub, and it is on GitHub I check if things are working as they should.